napari/_qt/dialogs/preferences_dialog.py

```diff
@@ -278,33 +278,15 @@ def build_page_dialog(self, schema, values):
         values : dict
             Dictionary of current values set in preferences.
         """
-        settings = get_settings()
         builder = WidgetBuilder()
         form = builder.create_form(schema, self.ui_schema)
 
-        # Disable widgets that loaded settings from environment variables
```

Comment:
@goanpeca, I ended up deleting this section.

The behavior is a bit different of course (but I think the original intent is met?) tell me what you think.  As I understand it, the goal was to prevent env var settings making it into the "permanent" yaml record.  That will happen with the PR as is, since anything that matches the env var will not be written to disk (see `test_settings_env_variables_do_not_write_to_disk` for details).

i guess I was thinking that it's ok if someone wants to change a setting during a napari session, even if was initiated from an env var... (i.e. this doesn't see env vars as "restricting" the possible values, just temporarily overriding them).
thoughts?